### ISSUE TITLE
Bug on Cosmos 1.5 `No such file or directory "/usr/local/aiflow/dags/dbt/dbt_venv/bin/python"`

### ISSUE THREAD

**Issue by tatiana** [astronomer/astronomer-cosmos#1096]
Two Astro customers reported that when updating to Cosmos 1.5, they started seeing the error:
```
No such file or directory "/usr/local/aiflow/dags/dbt/dbt_venv/bin/python"
```

As illustrated in the two screenshots below:

![Image](https://github.com/user-attachments/assets/148fc806-c112-42f6-ace3-958e2386e7fa)
![Image](https://github.com/user-attachments/assets/859eb89e-8157-4d93-8492-052f4a7b1500)

**Comment by tatiana** [comment]
After lots of time trying to reproduce and understand this issue, I narrowed it down:
- inside the customers dbt project folder, there was a symbolic link to a no longer existent file

Cosmos was raising an exception while trying to parse the file and create a version hash to represent the version of that dir.

I validated a possible solution with one of the customers earlier today, available in Cosmos 1.5.1a3 - and they confirmed it solved the problem. I'll make a quick PR with the solution and unittests today

**Comment by tatiana** [comment]
We fixed the issue in #1097, and, in parallel to this, we would like to encourage users to have Python virtual environments outside the dbt project folder and outside of the Airflow DAGs folder.
